board/drivers/can.h

```diff
@@ -42,10 +44,14 @@ uint32_t ignition_can_cnt = 0U;
 #define ALL_CAN_SILENT 0xFF
 #define ALL_CAN_LIVE 0
 
-int can_live = 0, pending_can_live = 0, can_loopback = 0, can_silent = ALL_CAN_SILENT;
+int can_live = 0;
+int pending_can_live = 0;
+int can_loopback = 0;
+int can_silent = ALL_CAN_SILENT;
 
 // ********************* instantiate queues *********************
-
+// cppcheck-suppress misra-c2012-20.10
```

Comment:
this one is stupid imo, do we want to suppress it here, or add it to the global suppressions?

Reply:
I would add it to the globals as it makes no sense imho. "The # and ## preprocessor operators should not be used"

Reply:
this one is advisory, but I agree with the rationale and think we should enforce it